TestingApp/DeeplinkService/ConfigurationDeeplinkHandler.swift

```diff
@@ -0,0 +1,22 @@
+import UIKit
+
+struct ConfigurationDeeplinkHandler: DeeplinkHandler {
+    private let window: UIWindow?
+
+    init(window: UIWindow?) {
+        self.window = window
+    }
+
+    func handleDeeplink(
+        with path: String?,
+        queryItems: [URLQueryItem]?
+    ) -> Bool {
+        guard
```

Comment:
This bit of logic could be covered with a test.

Reply:
Added tests bedcfffdb2dc2d9b2c8f19c96d83ce597a5ecc97